dashboard/test/controllers/lessons_controller_test.rb

```diff
@@ -493,18 +474,12 @@ class LessonsControllerTest < ActionController::TestCase
     sign_in @levelbuilder
     Rails.application.config.stubs(:levelbuilder_mode).returns true
 
-    script = create :script
-    lesson_group = create :lesson_group, script: script
-    lesson = create :lesson, script: script, lesson_group: lesson_group
-    lesson_activity = create :lesson_activity, lesson: lesson
-    activity_section = create :activity_section, lesson_activity: lesson_activity
+    unit = create :script, :with_lessons, lessons_count: 1
+    lesson = unit.lessons.first
     create(
       :script_level,
-      script: script,
       assessment: true,
-      activity_section: activity_section,
-      activity_section_position: 1,
-      lesson: lesson,
+      activity_section: lesson.activity_sections.first,
```

Comment:
this is one way to get a custom script level -- use `with_lessons`, then add your own script_level.